lib/ruby_lsp/requests/formatting.rb

```diff
@@ -67,16 +92,10 @@ def run
 
       sig { returns(T.nilable(String)) }
       def formatted_file
-        case @formatter
-        when "rubocop"
-          if defined?(Support::RuboCopFormattingRunner)
-            Support::RuboCopFormattingRunner.instance.run(@uri, @document)
-          end
-        when "syntax_tree"
-          Support::SyntaxTreeFormattingRunner.instance.run(@uri, @document)
-        else
-          raise InvalidFormatter, "Unknown formatter: #{@formatter}"
-        end
+        formatter = Formatting.formatters[@formatter]
+        raise InvalidFormatter, "Formatter is not available: #{@formatter}" unless formatter
```

Comment:
Will this error with `Formatter is not available: none` if `@formatter = "none"`?

Reply:
It won't because we return early in the `run` method.